### PR TITLE
Update use-cases page

### PR DESCRIPTION
**Pull request by brainwave0**
This implements the changes for issue #419.

### CODE REVIEW

File: docs/concepts/usage-ideas-examples.md [hunk 1]
```diff
@@ -94,7 +90,11 @@ Thanks to recent improvement with IPFS Cluster, you can call for help to store y
...
+Let your users stream from each other to remove the need for a server without overloading a popular streamer. You can get inspired by the [working prototype](https://github.com/tomeshnet/ipfs-live-streaming) of Toronto Mesh, or by the [experimentations](https://blog.fission.codes/experimenting-with-hls-video-streaming-and-ipfs/) of [Fission](https://fission.codes), a heavy player of the IPFS ecosystem. [Fleek](https://fleek.co) (previously known as Terminal.co) also did some [experimentation](https://medium.com/temporal-cloud/introducing-s3x-endless-ipfs-dynamic-possibilities-stream-videos-host-dynamic-websites-f0072127070f) using their Amazon S3/IPFS compatibility tool.
```
Comment (johnnymatthews) [inline docs/concepts/usage-ideas-examples.md:93]:
```suggestion
Let your users stream from each other to remove the need for a server without overloading a popular streamer. You can get inspired by the working prototype of [Toronto Mesh](https://github.com/tomeshnet/ipfs-live-streaming), or by the [experimentations](https://blog.fission.codes/experimenting-with-hls-video-streaming-and-ipfs/) of [Fission](https://fission.codes). [Fleek](https://fleek.co) also did some [experimentation](https://medium.com/temporal-cloud/introducing-s3x-endless-ipfs-dynamic-possibilities-stream-videos-host-dynamic-websites-f0072127070f) using their Amazon S3/IPFS compatibility tool.
```

File: docs/concepts/usage-ideas-examples.md [hunk 2]
```diff
@@ -148,7 +150,7 @@ If you are using Amazon Web Service's S3 connections for your cloud, you can use
 
 ### NextCloud integration
 
-JusticeNode has created [an extention](https://github.com/justicenode/files_external_ipfs) for the popular self-hosted cloud service NextCloud. This integration allows users to use IPFS as external storage.
+JusticeNode has created [an Extension](https://github.com/justicenode/files_external_ipfs) for the popular self-hosted cloud service NextCloud. This integration allows users to use IPFS as external storage.
```
Comment (johnnymatthews) [inline docs/concepts/usage-ideas-examples.md:153]:
```suggestion
[JusticeNode has created an extension](https://github.com/justicenode/files_external_ipfs) for the popular self-hosted cloud service NextCloud. This integration allows users to use IPFS as external storage.
```

**Comment by brainwave0** [comment]
Yes, they still use IPFS.

**Comment by johnnymatthews** [comment]
Perfect! Thanks for running the research on that @brainwave0!

**Pull request merged**